CONTRIBUTING.md

```diff
@@ -95,11 +95,18 @@ Once you've matched a function, run the following:
 
 Then, go ahead and [create a pull request](https://github.com/pmret/papermario/pulls)!
 
-## NON_EQUIVALENT
+## Trouble matching a function
 
-If you make significant progress on a function but are unable to 100% match it, feel free to wrap it in a NON_EQUIVALENT check:
+Sometimes when working on a function, for various reasons, you'll be unable to match it. The following two sections describe what labels we give these functions and when. Any time you wrap a function in a label like this, please leave a comment above the function with a brief description of the issue(s).
+
+### NON_EQUIVALENT
+
+If you are working on a function but can't figure out how to tackle certain parts, or you're sure that it's not equivalent, you can wrap it in NON_EQUIVALENT.
+
+Please make a best-effort attempt to any function you put in a PR, though. Half-finished functions aren't necessarily much use to the next decomper.
```

Comment:
Probably missing the word "finish" here

Reply:
no, to any. what part of that don't you understand

Reply:
lol yeah good catch

Reply:
lol